javalin/src/main/java/io/javalin/core/routing/PathSegment.kt

```diff
@@ -45,8 +45,12 @@ sealed class PathSegment {
         override fun asRegexString(): String = regex
         override fun asGroupedRegexString(): String = groupedRegex
     }
+
 }
 
+fun createSlashIgnoringParam(string: String) = PathSegment.Parameter.SlashIgnoringParameter(string)
```

Comment:
Maybe mark these functions as internal? Can't see anyone using them outside of this context.

Reply:
Good suggestion, will do!